extension/indexer/accepted_subscriber.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package indexer
+
+import (
+	"context"
+
+	"github.com/ava-labs/hypersdk/chain"
+)
+
+type AcceptedSubscriber interface {
+	Accepted(ctx context.Context, blk *chain.StatelessBlock) error
+}
+
+type CombinedAcceptedSubscriber struct {
+	subscribers []AcceptedSubscriber
+}
+
+func NewCombinedAcceptedSubscriber(subscribers ...AcceptedSubscriber) *CombinedAcceptedSubscriber {
```

Comment:
Can we call this function + this type `func NewAcceptedSubscribers(...) *AcceptedSubscribers`?

Reply:
Updated